http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/CommonTestData.scala

```diff
@@ -287,6 +291,21 @@ object CommonTestData {
     instance.copy(key = modifyInstanceKey(instance.key), googleId = instance.googleId + 1)
   def modifyInstanceKey(instanceKey: DataprocInstanceKey): DataprocInstanceKey =
     instanceKey.copy(name = InstanceName(instanceKey.name.value + "_2"))
+
+  def makePersistentDisk(id: Int): PersistentDisk = PersistentDisk(
```

Comment:
shall we start making scalacheck generators instead?

Reply:
Hmm.. yeah maybe